### PR TITLE
Add key vault certificate capability

### PR DESCRIPTION
**Pull request by javgallegos**
Signed-off-by: Javier Gallegos <jgallegosjr22@gmail.com>

### Description

Adds the ability to specify a certificate stored in Azure KeyVault and add to an Azure VM.

### Issues Resolved

- http://blog.baowebdev.com/2016/07/installing-a-certificate-from-azure-keyvault-into-an-azure-vm/

### Check List

- [ ] New functionality includes tests
- [ ] All tests pass
- [ ] PR title is a worthy inclusion in the CHANGELOG

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,11 +1,10 @@
 ### Description
 
-[Please describe what this change achieves]
+Adds the ability to specify a certificate stored in Azure KeyVault and add to an Azure VM.
```
Comment (stuartpreston) [inline .github/PULL_REQUEST_TEMPLATE.md]: Please could you remove this file from your PR? The files in .github/ are internal helpers for the GitHub UI

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # kitchen-azurerm Changelog
 
+## [0.14.10] - 2019-10-21
```
Comment (stuartpreston) [inline CHANGELOG.md]: Please remove this file from the PR, I'll deal with the Changelog at release time. Thanks!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (javgallegos) [reply .github/PULL_REQUEST_TEMPLATE.md]: Sure thing.

File: CHANGELOG.md (hunk 2 above)
Comment (javgallegos) [reply CHANGELOG.md]: Definitely.

**Pull request closed without merging**